prov/efa/src/efa_strerror.c

```diff
@@ -101,6 +102,22 @@ static const char *efa_errno_str(enum efa_errno err)
 		: errno_str[err - FI_EFA_ERRNO_OFFSET];
 }
 
+static const char *efa_debug_str(enum efa_errno err)
+{
+	switch (err) {
+	case FI_EFA_REMOTE_ERROR_BAD_DEST_QPN:
+		return "\nThis error can be caused by a crashed peer. "
+			   "Please verify the receiver status";
+	case FI_EFA_LOCAL_ERROR_UNRESP_REMOTE:
```

Comment:
Does `FI_EFA_LOCAL_ERROR_UNRESP_REMOTE` necessarily mean problems with security group rules? Could it not be other network failures?

Reply:
Misconfigured ingress/egress rule is a common/basic cause. There could be other reasons - the hint states the obvious to remind the user to avoid those traps.

Reply:
In that case, the error message should specify that problems with SG rules is a common but just one of the possible root causes. As it is, the message is misleading.

Reply:
Thank you will address!

Do you have a handy rephrase in mind?

Reply:
Your new version looks good to me!